alembic/versions/94e1ae982b64_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: 94e1ae982b64
+Revises: c66b6b222bde, d2f22cbd4c5a
+Create Date: 2022-02-01 01:01:45.400770+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "94e1ae982b64"
+down_revision = ("c66b6b222bde", "d2f22cbd4c5a")
```

Comment:
This was the recommended fix in stackoverflow.

Reply:
That's if there was actually a prior branch, but there wasn't in our case. That was artificially introduced by the order in which you ran the revisions. The correct thing to put in this field is just "d2f22cbd4c5a"

Reply:
https://stackoverflow.com/questions/22342643/alembic-revision-multiple-heads-due-branching-error

Reply:
Yes. I see that. It might make sense in some contexts, like if you've accidentally deployed something like this on site. But we haven't, so there's no reason to introduce this complexity into our database revision history.

Reply:
I can fix it and test the fix on my machine (with upgrades and downgrades in alembic) to make sure it's behaving if you want me to.

Reply:
I think it is fixed.  I made yours downgrade from mine.

Reply:
Maybe you can test the alembic before we deploy to site?

Reply:
Sure, I'd be happy to. I think what you have is correct, it is an upgrade on top of the antenna indices change, so the version history is linear now. But I'll put it through some testing on my machine.

Reply:
I tested it and it's fine.

Reply:
Thanks - as soon as it clears I'll deploy on site.